### ISSUE TITLE
Should wasm2c be a devDependency instead?

### ISSUE THREAD

**Issue by juj** [emscripten-core/emscripten#19367]
In PR #11213 @kripken added a new NPM repository dependency `wasm2c` into our NPM manifest.

Are there relevant users of this NPM package who are not Emscripten developers?

I.e. should `wasm2c` instead be listed under `devDependencies` and not `dependencies`?

In other words, is wasm2c only used by developers who are developing Emscripten itself, or is it a "end user facing" feature for any Emscripten user?

**Comment by kripken** [comment]
It is end user facing in that any end user that uses `-sWASM2C` will end up needing it.

However, I think last I talked with @sbc100 about this it seemed like maybe we should just expect the user to have `wasm2c` in their path. So we should be able to remove this eventually.

**Comment by sbc100** [comment]
Do we even want to continue to support `-sWASM2C`?  Last time we talking I thought we should just remove it completely since it was mostly there as a experiment.

**Comment by kripken** [comment]
I feel it could still be useful. There isn't another tool that easily compiles to sandboxed C code that I am aware of, without manual work. But that might not be reason enough to keep it, so I wouldn't object to removing it.

**Comment by juj** [comment]
Yeah, I don't oppose having it, if its intent was to be for Emscripten's end users (== npm's `dependencies` packages), and not just for people who develop Emscripten itself (== npm's `devDependencies' packages). Though we don't use it at Unity, and I noticed it takes up about 1.5 megs of space, so I configured it in our redistributables to be deleted.

Just thought to double check if it was deliberately placed in the intended package category in the first place.
